features/dd-sdk-android-trace/build.gradle.kts

```diff
@@ -50,8 +50,14 @@ android {
     }
 }
 
+repositories{
+    mavenLocal()
+}
+
 dependencies {
     api(project(":dd-sdk-android-core"))
+    api("io.opentelemetry:opentelemetry-api:1.4.0")
+    implementation("com.datadoghq:dd-java-agent:1.28.0-SNAPSHOT")
```

Comment:
why `SNAPSHOT`?

Reply:
it is a local maven version that I modified for this project. In APM they are following the same approach for local/dev releases so they add a `SNAPSHOT` at the end.